projects/dsp-ui/src/lib/viewer/values/text-value/text-value-as-xml/text-value-as-xml.component.ts

```diff
@@ -0,0 +1,209 @@
+import { Component, Inject, Input, OnChanges, OnDestroy, OnInit, SimpleChanges } from '@angular/core';
+import { BaseValueComponent } from '../../base-value.component';
+import { Constants, CreateTextValueAsXml, ReadTextValueAsXml, UpdateTextValueAsXml } from '@dasch-swiss/dsp-js';
```

Comment:
Imports need to be organized

Reply:
done in https://github.com/dasch-swiss/dsp-ui-lib/pull/134/commits/2e7b3456557533e626891e138655faa23a444808